docs/usecases/RStudio_user.rst

```diff
@@ -0,0 +1,207 @@
+.. \_usecase_Rstat:
+
+DataLad and RStudio: First steps
+---------------------------
+
+.. index:: ! Usecase; R users quickstart
+
+This use case sketches typical entry points for R and `Rstudio <https://en.wikipedia.org/wiki/RStudio>`_ users. 
+
+#. A repository having submodules for data and code is cloned.
+#. R scripts are developed in Rstudio
+#. R scripts are run using ``datalad run``
+
+(This is a `hello world` type of analysis, used only for demonstration purposes.)
+
+The Challenge
+^^^^^^^^^^^^^
+
+Max has been using Rstudio together with :term:`GitHub` for a long time. They know how :term:`Git`
+works. Max has learned that Git will not work with their new project,
+because there will be too many files and some dataset files will be too large.
+Max read the DataLad :ref:`Handbook Basics <basics-intro>` and is decided to use DataLad.
+They indeed want to version control larger files, and split files in several repositories, linked as "DataLad dataset hierarchies".
+Max still want to use Rstudio and a combination of R and python scripts for the
+data analysis.
+
+Bobby is a data manager who already learned (the hard way), how to handle datalad
+using Rstudio. They have also created a :term:`GIN` repository with :term:`submodule`\s 
+for data and for code, using the [Tonic tool and templates](https://gin-tonic.netlify.app).
```

Comment:
This statement is primarily emotion (which is OK). However, instead of merely stating that there is also a "GIN" and a "Tonic" part of the scenario, it would be better to say "why".

GIN likely is here, because it could host large files. If that is the is main reason, it should make that explicit. If so, it would be useful to say that GIN is one convenient solution, but not the only one.

Likewise the role of Tonic should be clarified. Right now it seems that it is needed to set up the GIN repos.

Given that the main content of this use case is about executing code locally (and not about pushing the results somehere), it is worth considering moving this information into a box, to avoid the impression that it is necessary to have these for running RStudio with DataLad.